src/SerilogWeb.Classic/Classic/ApplicationLifecycleModule.cs

```diff
@@ -69,9 +72,17 @@ public static void Register()
         }
 
         /// <summary>
+        /// You can add predicates to this list and they will be evaluated before
+        /// logging the request. If *any* fail the request will not be logged.
+        /// </summary>
+        public static ConcurrentBag<Func<HttpContext, bool>> RequestFilterPredicates => _requestFilterPredicates;
```

Comment:
`RequestFilter`/`RequestFilters` would be a cleaner name, I think `Predicate` is Hungarian here.